suse_migration_services/units/ssh_keys.py

```diff
@@ -57,22 +58,24 @@ def main():
         sshd_config_host_keys_entries = []
         log.info('Copying host ssh keys')
         for system_ssh_host_key in system_ssh_host_keys:
-            shutil.copy(system_ssh_host_key, '/etc/ssh/')
+            if not system_ssh_host_key.endswith('ssh_host_key'):
```

Comment:
But host keys don't end with "ssh_host_key" keys are named according to the encryption algorithm being used such as "ssh_host_ecdsa_key".

I am confused

Reply:
There is one key that is encrypted and not PEM encoded named /etc/ssh/ssh_host_key. That one can't be copied